content/hi/automation.md

```diff
@@ -0,0 +1,12 @@
+---
+टाइटल: ऑटोमेशन 
+---
+## ऑटोमेशन का क्या मतलब है ?
+ऑटोमेशन का अर्थ स्वचालन हैं । यह एक तकनीक है जिसमे किसी भी मशीन को फुल्ली आटोमेटिक बनाया जाता हैं आसान भाषा में बताया जाए तो ये एक लेबर सेविंग टेक्नोलॉजी हैं जिसमे किसी भी काम को करने के लिए कम से कम लोगो की आवशकता होती हैं, उद्धहरण के लिए जैसे : चैटबॉट , जब भी आप कोई वेबसाइट पर विजिट करते है तो आपके पास एक हेलो पॉपअप होता है तो वो एक ऑटोमेशन है | आजकल ऑटोमेशन हर क्षेत्र में प्रयोग किया जाता है | 
+
+## ऑटोमेशन क्या संकट का समाधान लाता हैं ?
+ऑटोमेशन प्रबंधनो और ऑपरेटरों को एनवायरनमेंट मैनेज यानी की पुराने मैन्युअल ऑपरेशन्स को ऑटोमेट करने में मदद करता है | उद्धहरण के तोर पर हम चैटबॉट समझाते है - पहले वेबसाइट के मालिकों को बहुत से अधिकारियों को काम पर रखना पड़ता था जो की सिर्फ वेबसाइट पर आने वाले लोगो की तकलीफे सुने , पर ऑटोमेशन के जरिये ये काम आसान होगया है - जिससे की प्रोडक्ट की कॉस्ट में भी गिरावट हुई है | 

```

Comment:
- तोर is wrong, should be तौर 
- जरिये is wrong, should be ज़रिये
-  space in between होगया
- कॉस्ट can be replaced with लागत